src/style.css

```diff
@@ -0,0 +1,146 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.container {
+  height: 406px;
+  width: 198px;
+    
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.photo {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  padding-top: 33px;
+  margin: 0 auto 40px;
+}
+
+.content {
+  display: inline-flex;
+  flex-direction: column;
+  gap: 4px;
+  padding-left: 17px;
+}
+
+.description {
+  margin: 0;
+}
+
+.content__description {
+  font-family: inherit;
+  
+  font-size: 12px;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
+  font-weight: 500;
+
+  color: #060b35;
+}
+
+.code {
+  margin: 0;
+}
+
+.content__code {
+  font-family: inherit;
+  
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  padding: 16px 16px 0;
+  margin: 0;
+}
+
+.rating_stars {
+  display: flex;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.rating_review {
+  font-weight: 400;
+  width: 53px;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  padding: 24px 16px 16px;
+}
+
+.price__in-words {
+  width: 32px;
+  height: 18px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.price__in-numbers {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.purchase {
+  padding: 0 16px 17px;
+}
+
+.purchase__button {
+  width: 166px;
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  color: #fff;
+  text-transform: uppercase;
+  cursor: pointer;
+    
+  background: #00acdc;
+  border: #00acdc;
+  border-radius: 5px;
+}
+
+.purchase__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
You should set this border to the button element and change only border-color on hover state